packages/mcdoc/src/runtime/checker/index.ts

```diff
@@ -913,6 +913,10 @@ function simplifyIndexed<T>(
 	for (const index of typeDef.parallelIndices) {
 		let lookup: string[] = []
 		if (index.kind === 'static') {
+			if (index.value === '%fallback') {
+				values = child.fields.filter(f => f.kind === 'pair').map(f => f.type)
+				break
+			}
```

Comment:
I am already re-introducing this behavior in #1370, which also does some additional changes for performance, so the mcdoc/runtime changes should not be done in this PR to avoid conflicts.

Reply:
Sounds good, I'll revert this